backend/src/main/java/com/bakdata/conquery/models/concepts/Connector.java

```diff
@@ -170,6 +172,10 @@ public synchronized void addImport(Import imp) {
 		}
 	}
 
+	public static boolean isNotContained(int[] mostSpecificChildren) {
+		return Arrays.equals(mostSpecificChildren, NOT_CONTAINED);
```

Comment:
Nur für den Fall, kannst du bitte checken, dass die `-1` nie zusammen mit anderen Werten in einem Array vorkommt?